src/hotspot/share/runtime/os.cpp

```diff
@@ -1341,14 +1341,57 @@ void os::print_location(outputStream* st, intptr_t x, bool verbose) {
     for (address p = addr; p < align_up(addr + 1, sizeof(intptr_t)); ++p) {
       st->print(" %02x", *(u1*)p);
     }
+#ifndef PRODUCT
+    // corruption or uninitialized access pattern
+    const intptr_t pat = 0xabababababababab;
```

Comment:
How about constructing this from `badResourceValue`?

Reply:
We could also put another variable centrally next to badResourceValue , see 
https://github.com/openjdk/jdk/blob/4fc6d4135e795d18a024a6035908f380b81082d1/src/hotspot/share/utilities/globalDefinitions.hpp#L1044
do you have a good naming for this ? Maybe `badZapValue` ?

Reply:
Maybe badResourceValueWord?  But I would put it in os.cpp instead of globalDefinitions.hpp until there are other clients.

Reply:
Sounds good , let's use this naming !